cypress/support/pages/homeCatalogue.pageObject.js

```diff
@@ -17,6 +17,84 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  clickOnButton(buttonName) {
+    cy.contains('.btn', buttonName)
+      .click();
+  }
+
+  verifyAddingToCart() {
+    return this.assertAllert('Product added');
```

Comment:
There seems to be a typo in the method name `assertAllert`. It should likely be `assertAlert` if you are trying to assert an alert message.